frontend/alert/src/components/managealert/index.js

```diff
@@ -0,0 +1,189 @@
+import React, { useState, useEffect } from "react";
+import AwesomeDebouncePromise from "awesome-debounce-promise";
+import { ManageAlert, ManageAlertHeader } from "./ManageAlertUI";
+import { AlertStatus, AlertRepeat, AlertAction } from "./AlertItemEnums";
+import getCsrf from "../../csrf";
+
+const fetchAlerts = () => (
+    fetch("/api/alert/api/registrations")
+        .then(res => res.json())
+);
+
+const processAlerts = (setAlerts) => {
+    const fetchPromise = () => (
+        fetchAlerts()
+            .then(res => (
+                res.map((section) => {
+                    let datetime = null;
+                    if (section.notification_sent) {
+                        const date = Intl.DateTimeFormat("en-US")
+                            .format(new Date(
+                                section.notification_sent_at
+                            ));
+                        const time = Intl.DateTimeFormat("en-US", {
+                            hour: "numeric",
+                            minute: "numeric",
+                            hour12: true,
+                        }).format(new Date(section.notification_sent_at));
+                        datetime = `${date} at ${time}`;
+                    }
+
+                    const status = section.section_status
+                        ? AlertStatus.Open : AlertStatus.Closed;
+
+                    let repeat;
+                    if (section.is_active) {
+                        if (section.auto_resubscribe) {
+                            repeat = AlertRepeat.EOS;
+                        } else {
+                            repeat = AlertRepeat.Once;
+                        }
+                    } else {
+                        repeat = AlertRepeat.Inactive;
+                    }
+
+                    return {
+                        id: section.id,
+                        original_created_at: section.original_created_at,
```

Comment:
this would be a good spot to transition from python snake_case to JS camelCase.
```suggestion
                        originalCreatedAt: section.original_created_at,
```

Reply:
I'm guessing this would require some find-and-replace in other places